src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -757,6 +757,9 @@ namespace Opm {
                     auto& well = this->m_wells.at(well_name);
                     bool switching_from_injector = !well.isProducer(currentStep);
                     WellProductionProperties properties(well.getProductionProperties(currentStep));
```

Comment:
Only very little information can be transferred through report steps, so making a copy is of very limited usage comparing with creating a new one.

Reply:
You are probably right - but that is unfortunate; as the 'carry information forward' idiom is quite central to the Schedule implementation.